test/libsolidity/smtCheckerTests/array_members/push_as_lhs_3d.sol

```diff
@@ -25,5 +25,7 @@ contract C {
 		assert(c[length1 - 1][length2 - 1][length3 - 1] == 200);
 	}
 }
+// ====
+// SMTIgnoreCex: yes
 // ----
-// Warning 6328: (570-625): CHC: Assertion violation happens here.\nCounterexample:\nc = [[[2]]]\n\nTransaction trace:\nC.constructor()\nState: c = []\nC.g()
+// Warning 6328: (570-625): CHC: Assertion violation happens here.
```

Comment:
Happens sometimes. In order to get counterexamples we need to run the solver without preprocessing, so sometimes it can prove that something is unsafe (with preprocessing) but can't give a counterexample. Any change in the encoding can change this result